cmd/app/crawler.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/PuerkitoBio/goquery"
+	"net/http"
+	"net/url"
+	"regexp"
+	"strings"
+	"sync"
+	"time"
+)
```

Comment:
use `go import`

Reply:
yes, firstly standards libs and after from another source
like this
<img width="729" alt="Screenshot 2022-06-27 at 16 23 38" src="https://user-images.githubusercontent.com/29044140/175952292-a280244d-7cd4-4131-a623-f744188f2468.png">

Reply:
OK, done. Thnx.